client/stories/3-admin/1-approval-chains.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta } from "@storybook/addon-docs"
+import approvalFlowDiagram from "./1-approval-chains.stories.rules-flow.drawio.svg"
+
+<Meta title="3. ANET Admin Stories/1. Approval chains" />
+
+# Approval Chains
+
+On report publication, or planned engagement approval, the corresponding approval chain initiated.
+Approval chains are configures in organizations.
+By editing an organization, a user can also edit the approval chains.
+Each organization can define 2 types of approval processes:
+1. Engagement planning approval
+2. Report publication approval
```

Comment:
Can simply be `1. Report publication approval`: let MarkDown do the numbering for you.